app/javascript/flavours/glitch/components/status.js

```diff
@@ -251,10 +251,15 @@ class Status extends ImmutablePureComponent {
     // as it could cause surprising changes when receiving notifications
     if (settings.getIn(['content_warnings', 'shared_state']) && status.get('spoiler_text').length && !status.get('hidden')) return;
 
+    let autoCollapseHeight = parseInt(autoCollapseSettings.get('height'));
+    if (status.get('media_attachments').size && !muted) {
+      autoCollapseHeight += 210;
```

Comment:
The actual computed height of the media gallery in advanced mode is 212px.
In simple mode, media-gallery height varies widely.

Default value was 250; dropping this to 210 gives a user experience closer to expectations.

Reply:
I'm not opposed to this change, but it's also browser specific and even timeline specific.
For example in chrome it is 212px in home, but 209px in the local timeline.
In Firefox it is 219px in home and 216px in the local timeline.

Reply:
Either way it sounds like 250 isn't the right value. To my mind, taking the lower end of the range makes sense; I've enabled this setting to clean up my timeline, presumably. Being on the aggressive side of collapsing makes sense for the user intent.

Reply:
I agree.
Would be good to change the comment above the change to reflect the change.

```js  
//   -  The user has decided to collapse long statuses and the status is
//      over 400px (without media, or 650px with).
```

to

```js
  //   -  The user has decided to collapse long statuses and the status is
  //      over the user set value (400px by default without media, or 610px with).
```

Other than that I don't see anything that would prevent merging this.